web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/forvaltning/stonadsstatistikk/StønadsstatistikkMigreringTask.java

```diff
@@ -60,30 +62,28 @@ class StønadsstatistikkMigreringTask implements ProsessTaskHandler {
 
     @Override
     public void doTask(ProsessTaskData prosessTaskData) {
-        var fagsakId = Long.parseLong(prosessTaskData.getPropertyValue(FAGSAK_ID));
-
-        LOG.info("Produserer stønadsstatistikk for fagsak {}", fagsakId);
-        var alleVedtak = finnAlleVedtakForFagsak(fagsakId);
-        if (alleVedtak.isEmpty()) {
-            LOG.info("Produserer stønadsstatistikk ingen gjeldende vedtak for fagsak {}", fagsakId);
-        } else {
-            LOG.info("Fant {} vedtak på fagsak {}", alleVedtak.size(), fagsakId);
-        }
-        alleVedtak.stream().sorted(Comparator.comparing(BaseCreateableEntitet::getOpprettetTidspunkt)).forEach(this::produser);
+        var fom = LocalDate.parse(prosessTaskData.getPropertyValue(DATO_KEY), DateTimeFormatter.ISO_LOCAL_DATE);
+        LOG.info("Publiser migreringshendelse for vedtak opprettet {}", fom);
+
+        var vedtak = finnAlleVedtakForDato(fom);
+
+        LOG.info("Publiser migreringshendelse for {} vedtak", vedtak.size());
+        vedtak.forEach(this::produser);
```

Comment:
Dere kan jo diskutere om dere først vil lage en liste av dtos og så sende alle - vi spør om de tåler duplikater i morgen. High score er 8/9-2020 med 8149 vedtak, Neste er ompostering av SVP-ferie med 5179 vedtak.

Reply:
Enig, sikkert best at vi produserer dtos først. Mindre sjans for exception.
8000 vedtak på en task kan uansettkanskje bli mange? Kan evt vurdere 2 tasks per dag

Reply:
Hvordan splitte dagen? De 8149 var ifm G-regulering og veldig mange skjedde i løpet av en time ....
Dette går nok greit DB-messig - det er mest det å lagre mer enn 1000 tasks som er tungt (ikke-lineær tid)
Det er mer et stort volum som re-sendes på kafka.
Eller skal man se på task pr vedtak (eller med 10 vedtak.id som parameter) dersom antall vedtak for en dag > 1000/2000